docker-compose.yml

```diff
@@ -54,6 +55,32 @@ services:
       - "GF_SECURITY_ADMIN_USER=admin"
       - "GF_SECURITY_ADMIN_PASSWORD=my_adm1n_password"
 
+  web-view:
+    build: ./web-view
+    networks:
+      - backend
+    depends_on:
+      - database
+    environment:
+      # default database user
+      - "SQL_USER=aimlac"
+      - "SQL_PASS=aimlac"
+      - "SQL_HOST=aimlac-database"
+      - "SQL_DATABASE=llanwrytd"
+      - "JWT_SECRET=${JWT_SECRET}"
+    hostname: "aimlac-web-view"
+  
+  nginx:
+    image: nginx:1.21.6-alpine
+    networks:
+      - backend
+    depends_on:
+      - web-view
+    ports:
+      - 80:80
+    volumes:
+      - "${PWD}/web-view/nginx:/etc/nginx/conf.d"
```

Comment:
See my messages on discord about this.

Reply:
👍 , will move the nginx config into this repo